scripts/cleanup.sh

```diff
@@ -10,12 +10,7 @@ rm -rf /var/cache/yum
 
 # Clean up files to reduce confusion during debug
 rm -rf \
-    /var-old \
-    /var/log-old \
-    /var/log/audit-old \
-    /home-old \
```

Comment:
We still seem to create those partitions no? Don't we need the cleanup still?

Reply:
Forgot to remove their creation - resolved in https://github.com/aws-samples/amazon-eks-custom-amis/pull/58/commits/728aa0420881a3ef21de25f6cc551592faab977a